demisto_sdk/commands/common/content/objects/pack_objects/modeling_rule/modeling_rule.py

```diff
@@ -211,6 +211,10 @@ def __init__(self, path: Union[Path, str]):
         self._rules: List[SingleModelingRule] = []
         self.rules_dict: dict = {}
 
+    def __repr__(self) -> str:
+        path_name = Path(f"{self.path}").name
```

Comment:
```suggestion
        path_name = Path(self.path).name
```
Path can also take in a Path to its constructor, double check, i am sure in 99%
if its not, then keep it this way :)